app/src/main/java/org/wikipedia/analytics/AppearanceChangeFunnel.java

```diff
@@ -3,16 +3,19 @@
 import androidx.annotation.NonNull;
 
 import org.json.JSONObject;
+import org.wikipedia.Constants.InvokeSource;
 import org.wikipedia.WikipediaApp;
 import org.wikipedia.dataclient.WikiSite;
 import org.wikipedia.theme.Theme;
 
 public class AppearanceChangeFunnel extends Funnel {
     private static final String SCHEMA_NAME = "MobileWikiAppAppearanceSettings";
     private static final int REV_ID = 18113727;
```

Comment:
This needs to be a new revision ID, when Shay makes the schema changes.

Reply:
Done.